indica/readers/abstractreader.py

```diff
@@ -41,9 +43,9 @@ class DataReader(BaseIO):
 
     This defines the interface used by all concrete objects which read
     data from the disc, a database, etc. It is a `context manager
-    <https://docs.python.org/3/library/stdtypes.html#typecontextmanager>`_
+    <https://protect-eu.mimecast.com/s/f7vJCpzxoFzjOpcPXjtX?domain=docs.python.org>`_
```

Comment:
These URLs should be changed back, not sure where they lead but I don't have access to it so I suspect others won't either